.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,195 @@
+name: benchmarks
+
+env:
+  TARDIS_VER: release-2023.06.18
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled # benchmarks label required
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  build:
+    if: github.event_name == 'push' ||
+      contains(github.event.pull_request.labels.*.name, 'benchmarks')
```

Comment:
this has to be a pull_request_target event
the post comment step won't run if the event is pull_request.
I don't know that the labels thing would work with the pull_request_event however

Reply:
Changed it to 
```github.event_name == 'push' || github.event_name == 'pull_request'```